C/Savina/src/parallelism/PiPrecision.lf

```diff
@@ -39,11 +39,15 @@ preamble {=
 =}
 
-reactor Manager(numWorkers:int(20), scale:uint32_t({=5000ul=})) {
+reactor Manager(numWorkers:size_t=20, scale:uint32_t={=5000ul=}) {
+
+    // preamble {=
+    //     #include "gmp.h"
+    // =}
```

Comment:
```suggestion
```
Better to remove this completely. It looks like it is not needed, as the file level preamble included gmp.